apps/src/calc/calc.js

```diff
@@ -36,7 +36,9 @@ var _ = require('../utils').getLodash();
 var timeoutList = require('../timeoutList');
 
 var ExpressionNode = require('./expressionNode');
-var TestResults = require('../constants').TestResults;
+
+var ResultType = studioAppSingleton.ResultType;
```

Comment:
Good point. Wonder if it actually should be used - I think Im using true/false instead.